src/test/scala/chiselTests/BoringUtilsTapSpec.scala

```diff
@@ -573,6 +566,50 @@ class BoringUtilsTapSpec extends ChiselFlatSpec with ChiselRunners with Utils wi
     val verilog = circt.stage.ChiselStage.emitSystemVerilog(new Top(Definition(new Foo)))
   }
 
+  it should "work to rwTap an Instance[..]'s port" in {
+    import chisel3.experimental.hierarchy._
+    class UnitTestHarness extends Module {
+      val dut = Instantiate(new Dut)
+      probe.force(dut.widgetProbes.head, 0xffff.U)
+    }
+
+    @instantiable
+    class Widget extends Module {
+      @public val in = IO(Input(UInt(32.W)))
```

Comment:
should we test rwProbe of Input vs Output (or does the behavior not matter)?

Reply:
This is testing a rwTap of a RWProbe output port, the other tests in `BoringUtilsTap` tests rwTap's of inputs and outputs.

Reply:
oh yeah there is no such thing as RWProbe Input Port 👍